src/hooks/use-sync-states-progress-info.ts

```diff
@@ -183,14 +222,58 @@ export function useSyncStatesProgressInfo() {
 		[ __ ]
 	);
 
+	const getPushStatusWithProgress = useCallback(
+		( status: PushStateProgressInfo, response: ImportResponse ) => {
+			if ( status.key === pushStatesProgressInfo.creatingRemoteBackup.key ) {
+				const creatingRemoteBackupStep =
```

Comment:
Nit. Let's clarify that this variable refers to the "space/gap/range" available for this step in the progress bar. I think it's ok if we call the variable the same in each step.
```suggestion
				const progressRange =
```